multiworld/envs/mujoco/sawyer_xyz/sawyer_push_and_reach_env.py

```diff
@@ -228,15 +233,14 @@ def _set_puck_xy(self, pos):
         self.set_state(qpos, qvel)
 
     def reset_model(self):
-        if self.reset_counter % self.num_resets_before_hand_reset == 0:
-            self._reset_hand()
-        if self.reset_counter % self.num_resets_before_puck_reset == 0:
+        self._reset_hand()
```

Comment:
So we're going to reset the hand ever reset? I kind of liked having `num_resets_before_puck_reset` and `num_resets_before_hand_reset` separately. I don't feel strongly about it, but it just seems more explicit than the vague "`reset_free`" variable.

Reply:
yeah so the reason for that is if you don't reset the hand then the hand gets extended in weird positions that can be harder to recover from - its quite hard to restrict the joint angles to prevent this so I think a better solution is to always reset the hand